cms/djangoapps/contentstore/views/tests/test_library.py

```diff
@@ -54,7 +54,7 @@ def test_library_creator_status_with_is_staff_user(self):
     def test_library_creator_status_with_course_creator_role(self):
         _, nostaff_user = self.create_non_staff_authed_user_client()
         with mock.patch.dict('django.conf.settings.FEATURES', {"ENABLE_CREATOR_GROUP": True}):
-            add_users(self.user, CourseCreatorRole(), nostaff_user)
+            add_user_with_status_granted(self.user, nostaff_user)
```

Comment:
Is this a better/newer approach to granting the CourseCreator role to a user? It seems less clear to me. If you imported this method as `from course_creators.views import add_user_with_status_granted as grant_course_creator_status` then I would find it more clear.